src/containers/AuthenticationRoute/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Route, Redirect, withRouter } from "react-router-dom";
+
+class AuthenticationRoute extends Route {
+  isValidRouting = () => {
+    const { withAuth, isAuthenticated } = this.props;
```

Comment:
I think we could access isAuthenticated with mapStateToProps

Reply:
do You mean I have to create redux -store, initialize isAuthenticated there and get it through mapStateToProps?

Reply:
Please ignore this comment right now, but yes, eventually that's what we'll probably need to do